lib/portalUtils.js

```diff
@@ -50,7 +50,7 @@ _.extend(exports, {
     },
     getPom: function(c, p) {
         p = p || '';
-        var pomPath = path.join(c.config._local.path, p, '/pom.xml');
+        var pomPath = path.join(c.config.bbrc.path, p, '/pom.xml');
```

Comment:
What's `bbrc.path`? Path to where? I suppose to portal folder, but it's not clear at all.

Where those util methods are used - `getInfo()`, `getPom()`?